src/utils/text.scss

```diff
@@ -41,3 +41,32 @@
     }
   }
 }
+
+
+/**
+ * Font size utilities
+ */
+
+.u-fontSizeXs {
+  font-size: $tu-xsmall-fontSize !important;
+}
+
+.u-fontSizeSm {
+  font-size: $tu-small-fontSize !important;
+}
+
+.u-fontSizeBase {
```

Comment:
I kept this `base` vs `medium` as we did with spacing to preserve it as the base of our modular scale. I'm open to something else though if you disagree @williamdingwall.

Reply:
'Md' might make more sense with everything else.. seems like with 'Base' part of the class we're trying to describe usage too much instead of just providing a consistent scale of classes to work with